controllers/discordactions.js

```
@@ -118,6 +118,67 @@
+    if (!roleExists) {
+      return res.boom.notFound("Group role not found");
+    }
+
+    const updateData = {};
+    if (rolename) {
+      if (!rolename.startsWith("group-")) {
+        rolename = `group-${rolename}`;
+      }
+
+      const { roleExists: nameExists, existingRoles: nameExistingRoles } =
+        await discordRolesModel.isGroupRoleExists({ rolename });
```

ESLint

[error] Replace `⏎········await·discordRolesModel.isGroupRoleExists({·rolename` with `·await·discordRolesModel.isGroupRoleExists({⏎········rolename,⏎·····`

(prettier/prettier)